### PR TITLE
fixes #31, support federated metastore connection over SSH tunnel

### LINKED ISSUE #31: enable waggle dance to connect to federate using bastion host
_As a Sysops_
_I want to federate to a remote Metastore in another region_
_So that I can support cross-region data access use-case_

### PR DESCRIPTION
**Pull request by pradeepbhadani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 ## Variables
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
+| bastion_ssh_key_secret_name | Secret name in AWS Secrets Manager which stores private key to login to Bastions. Secret's key should be `private_key` and value should be stored as base64 encoded string. Max character limit for a Secret's value is 4096. | string | `` | no |
```
Comment (massdosage) [inline README.md]: Bastions -> bastions

Comment (massdosage) [reply README.md]: to login to -> used to log in to

Comment (massdosage) [reply README.md]: base64 -> a base64

Comment (massdosage) [reply README.md]: Secret's -> secret's

File: README.md [hunk 2]
```diff
@@ -25,6 +26,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | region | AWS region to use for resources. | string | - | yes |
 | remote_metastores | List of VPC endpoint services to federate Metastores in other accounts. | list | `<list>` | no |
 | secondary_vpcs | List of VPCs to associate with Service Discovery namespace | list | `<list>` | no |
+| ssh_metastores | List of federated Metastores to connect over SSH via Bastion. | list | `<list>` | no |
```
Comment (massdosage) [inline README.md]: connect over SSH -> connect to over SSH

Comment (massdosage) [reply README.md]: Bastion -> bastion

File: variables.tf [hunk 3]
```diff
@@ -129,6 +129,24 @@ variable "remote_metastores" {
   default     = []
 }
 
+#list of maps, example: [ {bastion-host="10.x.x.x", metastore-host="10.x.x.x", port="9083", prefix="pre1", user="my-unix-user", mapped-databases="test1,test2"}, {bastion-host="10.x.x.x", metastore-host="10.x.x.x", port="9083", prefix="pre1", user="my-unix-user", writable-whitelist="db1,test", mapped-databases="test1,test2"} ]
+variable "ssh_metastores" {
+  description = "List of federated Metastores to connect over SSH via Bastion."
```
Comment (massdosage) [inline variables.tf]: The comments I made above for the README apply to this file too (I guess this is where it's actually coming from?)

File: README.md (hunk 1 above)
Comment (pradeepbhadani) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (pradeepbhadani) [reply README.md]: fixed

**Pull request merged**